dask/highlevelgraph.py

```diff
@@ -961,10 +961,10 @@ def __dask_distributed_pack__(self, client, client_keys: Iterable[Hashable]) ->
                     "annotations": layer.__dask_distributed_annotations_pack__(),
                 }
             )
-        return dumps_msgpack({"layers": layers})
+        return {"layers": layers}
```

Comment:
This isn't important, but is there a msgpack/serialization reason for wrapping in a dict? Versus:
```suggestion
        return layers
```

Reply:
This is a good point. I think we can probably just pass the list of layers directly. @madsbk mentioned that serializing twice is blocking some folks, so I'm going to merge this as is and we can address the dict vs. list point in a follow-up PR

Reply:
The idea here was to make it easy to extend in the future but yes, changing to a list of layers is also fine.